test/acceptance/ReactRefreshLogBox.test.js

```diff
@@ -678,6 +678,35 @@ test('unterminated JSX', async () => {
   await cleanup()
 })
 
+test('Module not found', async () => {
+  const [session, cleanup] = await sandbox()
+
+  await session.patch(
+    'index.js',
+    `
+      import Comp from 'b'
+
+      export default () => {
+        return (
+          <div>
+            <Comp>lol</Comp>
+          </div>
+        )
+      }
+    `
+  )
+
+  expect(await session.hasRedbox(true)).toBe(true)
+
+  const source = await session.getRedboxSource()
+  expect(source).toMatchInlineSnapshot(`
+    "./index.js
```

Comment:
This test doesn't work on Windows:
![image](https://user-images.githubusercontent.com/616428/89377869-acd61e80-d6c0-11ea-9385-6abfb2969d8b.png)

Reply:
Should be fine now 💯